src/main/java/com/github/tomakehurst/wiremock/verification/diff/Diff.java

```diff
@@ -319,12 +308,12 @@ private DiffLine<String> addUrlSectionWithSpacer(
   }
 
   private void addMethodSection(List<DiffLine<?>> diffLineList) {
-    DiffLine<RequestMethod> methodSection =
+    DiffLine<Set<RequestMethod>> methodSection =
         new DiffLine<>(
             "HTTP method",
-            requestPattern.getMethod(),
-            request.getMethod(),
-            requestPattern.getMethod().getName());
+            new RequestMethodMatcher(requestPattern.getMethods()),
```

Comment:
It would be great if we could preserve the existing behaviour when only a single method is specified rather than it always appearing as a collection